### DISCUSSION TITLE
Help - after installation

### DISCUSSION THREAD

**Discussion by Willianbr07** [blackhatethicalhacking/TerminatorZ#29] [General]
I found some errors after run TerminatorZ:

Traceback (most recent call last):
  **File "/home/EU/.local/bin/lolcat", line 242, in <module>
    sys.exit(run())**
             ^^^^^
  **File "/home/EU/.local/bin/lolcat", line 230, in run
    lolcat.cat(sys.stdin, options)**
  **File "/home/EU/.local/bin/lolcat", line 123, in cat
    for line in fd:
  File "<frozen codecs>", line 322, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf3 in position 1629: invalid continuation byte**

and:

Testing  for XSS vulnerability...10
**curl: option : blank argument where content is expected
curl: try 'curl --help' or 'curl --manual' for more information**
 is not XSS vulnerable
Testing  for SSRF vulnerability...
 is not SSRF vulnerable
Testing  for XXE vulnerability...
**curl: option : blank argument where content is expected
curl: try 'curl --help' or 'curl --manual' for more information**
 is not XXE vulnerable
Testing  for Insecure Deserialization vulnerability...
**curl: option : blank argument where content is expected
curl: try 'curl --help' or 'curl --manual' for more information**
 is not insecure deserialization vulnerable
Testing  for Shellshock vulnerability...
**curl: option : blank argument where content is expected
curl: try 'curl --help' or 'curl --manual' for more information**
 is not vulnerable to Shellshock RCE
Testing  for RCE vulnerability...
**curl: option : blank argument where content is expected
curl: try 'curl --help' or 'curl --manual' for more information**
 is not RCE vulnerable
Testing  for CSRF vulnerability...
**curl: option : blank argument where content is expected
curl: try 'curl --help' or 'curl --manual' for more information**
 is not CSRF vulnerable

Can someone help me figure out why these errors are occurring?

**Comment by blackhatethicalhacking** [comment]
Hello!
Its normal errors, it depends on the target and link sometimes, so this error is what it means. It has to do with curl as you can see, and some URLs are not probably on the structure, so it skips them and moves to next ones.

Black Hat Ethical Hacking
Support
 
 <https://www.blackhatethicalhacking.com/>	
***@***.*** - www.blackhatethicalhacking.com <http://www.blackhatethicalhacking.com/>
 <https://www.facebook.com/secur1ty1samyth>   <https://www.linkedin.com/company/black-hat-ethical-hacking>   <https://twitter.com/secur1ty1samyth>   <https://www.youtube.com/channel/UC7-AsunT7zO-ny5-U8glqkw>   <https://www.instagram.com/blackhatethicalhacking/?hl=en>

**Reply by Willianbr07** [reply]
thank you very much!
